torch/_inductor/config.py

```diff
@@ -114,6 +114,9 @@ def autotune_remote_cache_default() -> Optional[bool]:
 # set to True to enable the back-to-back GEMM pass
 b2b_gemm_pass = False
 
+# Whether to enable masked vectorization for the tail_loop
+masked_vec = True
+
```

Comment:
Please move it to config.cpp and name it `config.cpp.enable_loop_tail_vec`.

Reply:
Done, thanks for your advice!